modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -530,6 +530,17 @@ internal void Normalize()
             return v + (vd / len * delta);
         }
 
+        /// <summary>
+        /// Moves this vector toward <paramref name="to"/> smoothly (fast to slow) by the <paramref name="delta"/> amount. The function is defined as <code>lerp(this, to, 1 - exp(-delta))</code>.
+        /// </summary>
+        /// <param name="to">The vector to move towards.</param>
+        /// <param name="delta">The amount to move towards by.</param>
+        /// <returns>The resulting vector.</returns>
+        public readonly Vector2 MoveTowardSmooth(Vector2 to, real_t delta)
+        {
+            return this.Lerp(to, Mathf.ExpM1(-delta));
```

Comment:
```suggestion
            return this.Lerp(to, -Mathf.ExpM1(-delta));
```

Not consistent with the C++ implementation.